loadtests/Makefile

```diff
@@ -12,7 +12,7 @@ INSTALL = ARCHFLAGS=$(ARCHFLAGS) ./venv/bin/pip install
 .env.install:
 	virtualenv ./venv
 	$(INSTALL) pexpect >> loadtest.log && \
-	$(INSTALL) configparser==3.3.0r2 >> loadtest.log && \
+	$(INSTALL) configparser==3.3.0post2 >> loadtest.log && \
```

Comment:
I recall that the 3.5.0b2 is not compatible with 3.3.0r2 nor 3.3.0post2 I am afraid.